packages/x-charts/src/internals/plugins/corePlugins/useChartDimensions/useChartAxisSize.selectors.ts

```diff
@@ -0,0 +1,33 @@
+import {
+  selectorChartRawXAxis,
+  selectorChartRawYAxis,
+} from '../../featurePlugins/useChartCartesianAxis/useChartCartesianAxisLayout.selectors';
+import { createSelector } from '../../utils/selectors';
+
+export const selectorChartLeftAxisSize = createSelector([selectorChartRawYAxis], (yAxis) =>
```

Comment:
Single selectors for each, as returning an object was breaking memoization and `fastObjectShallowCompare` didn't seem to work.

Reply:
This might be an issue in other areas if we are returning new objects/arrays each time it runs.